### ISSUE TITLE
add choices for how NVDA indicates that text is contained within a track change

### ISSUE THREAD

**Issue by ainekc** [nvaccess/nvda#12765]
### Is your feature request related to a problem? Please describe.
Currently, when reading a Word document with numerous track changes, or indeed when pressing "a" in browse mode to cycle through them, the fact the end tags for insertions and deletions create a bit of a cacophony. The end tags are "not inserted" and "not deleted", which appears logical, but has the side-effect of meaning you are constantly hearing the words "inserted" and "deleted" as part of those tags and it's quite demanding trying to mentally filter what's what.

To further illustrate, if you cycle through insertions and deletions in browse mode, every time you move from an insertion to a deletion or vice versa (which is to say, most key presses), you get told not only about the insertion and deletion but also that it's not its opposite, which is superfluous info.
### Describe the solution you'd like
I would love to see an option where inserted and deleted text could be denoted by the start tag only, and then indicated by pauses surrounding the text and the text spoken at a lower pitch. This would make following the changes so much easier when using speech. 
### Describe alternatives you've considered
You can turn the reading of track changes off entirely, but that's not what I'm after.

**Comment by feerrenrut** [comment]
Thanks for raising this @ainekc
To ensure everyone is one the same page, could you include a short example of the speech output from NVDA (you can copy from the speech viewer).

**Comment by ainekc** [comment]
Sorry for delay. Here's a [test file](https://docs.google.com/document/d/1ob4R-SveIyP8p0Qke0fI-Jqj747m1vJl/edit?usp=sharing&ouid=113458485496149344264&rtpof=true&sd=true), enable the reading of editor revisions to test it.

Speech history if I cycle through revisions in browse mode using the letter a goes:

inserted  Elephant
not inserted  deleted  Dog
inserted  not deleted  Octopus
not inserted  deleted  Fish
inserted  not deleted  Cow
not inserted  deleted  Horse

**Comment by feerrenrut** [comment]
Right, to clarify what is going on in the document, there was a list with
- Hello world
- Cat 
- Dog
- Fish
- Horse
- Sheep

Several change suggestions are made to make it become a list with:
- Hello world
- Cat 
- Elephant 
- Octopus 
- Cow
- Sheep

Then there is the inserted / deleted markers:
- `'inserted'` flags the start of added text
- `'not inserted'` the end of the added text is
- `'deleted'` the start of deleted text
- `'not deleted'` the end of deleted text

One thing that is confusing about this is that "inserted" come before "not deleted":
> deleted Dog inserted not deleted Octopus

This could be clearer as "deleted Dog not deleted inserted Octopus"

It's more obvious why you might need this when the text being inserted or deleted is longer.

Some UX ideas to make this clearer:
- As requested by @ainekc, use an alternative voice for inserted and deleted.
- Use an earcon to mark the start and end of deleted / inserted text.
- When there is a boundary for both inserted and deleted, just say the new state:
  - EG: Instead of "inserted Elephant not inserted deleted Dog not deleted" say "inserted Elephant and deleted Dog not deleted"


Here are more test cases as a starting point for system tests: https://codepen.io/reefturner/full/mdwNzOd
